src/components/Popover/README.md

```diff
@@ -415,6 +417,8 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 Popovers usually contain an [option list](https://polaris.shopify.com/components/lists-and-tables/option-list) or an [action list](https://polaris.shopify.com/components/actions/action-list), but can also contain other controls or content.
 
+A popover can contain many numerous types of content. Whether it's a menu, grid or something entirely different! When `aria-expanded` is applied to an element, `aria-haspopup` will default to `menu`. To assist screen readers you'll need to pass `ariaHaspopup` to `Popover`.
```

Comment:
A few thoughts here:

- I'd like to remove the "Whether it's..." fragment, since it doesn't reference other patterns or components.
- In reference to `aria-expanded` and `aria-haspopup`, it's [a bit more complicated](https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup). I think we could simplify this by saying something like, "The `ariaHaspopup ` prop adds`aria-haspopup="true"` to the button. This should only be used when the popover contains a complex feature like an ARIA menu, grid, listbox, or similar features."
- We typically want to use people-centered language, so we'd want to say "to assist people who use screen readers" instead of "to assist screen readers".

Let's chat about this one on Tuesday, and involve @sadiesaurus if she's available.

Reply:
Hi @sadiesaurus , what are your thoughts on this?

Reply:
Heya Andrew! I agree with Devon's points. I'll make a suggestion for this paragraph which you can feel free to edit.

Reply:
Here's my stab at a rewrite! Though, I'm not certain if I've accurately captured the context, so let me know if something is wrong or missing. 

```suggestion
Popovers can contain many types of content. If a popover contains a complex feature such as an ARIA menu, grid, listbox, or similar feature, then use the `ariaHaspopup` prop. This adds `aria-haspopup="true"` to the button, which is helpful information for people who use screen readers. 
```

Reply:
It looks great! I made one small adjustment since you can provide more values than "true" to `ariaHasPopup` (`boolean | 'false' | 'true' | 'menu' | 'listbox' | 'tree' | 'grid' | 'dialog'`)